etherman/types.go

```diff
@@ -66,3 +67,10 @@ type SequencedForceBatch struct {
 	Nonce       uint64
 	polygonzkevm.PolygonZkEVMForcedBatchData
 }
+
+// ForkID is a sturct to track the ForkID event.
+type ForkID struct {
```

Comment:
What do you think about merging this with state/forkid.go ? Maybe moving everything to the state or the etherman. (IMO this is more related to state)

Reply:
I see your point, but the problem is that the state.ForkID struct doesn't fit very well with the needs of the event. The synchronizer translates the info received from L1 to the infor required by the state. We are doing the same thing with other structs such as `etherman.SequencedBatch` and `state.Batch and state.VirtualBatch`